tests/src/test/java/org/eclipse/hono/tests/amqp/AmqpConnectionIT.java

```diff
@@ -107,6 +107,35 @@ public void testAutoProvisioning(final VertxTestContext ctx) {
                 .setHandler(ctx.completing());
     }
 
+    /**
+     * Verifies that the adapter rejects connection attempts from an unknown device for which auto-provisioning is
+     * disabled.
+     *
+     * @param ctx The test context
+     */
+    @Test
+    public void testConnectFailsIfAutoProvisioningIsDisabled(final VertxTestContext ctx) {
```

Comment:
not sure but don't we already have a test for an unknown device's connect attempt? This is basically the same thing, isn't it?

Reply:
Yeah, it _should_ be the same. The protocol adapter checks whether automatic provisioning is enabled. This checks that automatic provisioning is not requested from the device registry by the protocol adapter if it is disabled for the trust anchor used.